xformers/benchmarks/LRA/code/model_wrapper.py

```diff
@@ -134,6 +136,66 @@ def __init__(self, config, model_name):
             * ff_config["hidden_layer_multiplier"]
         )
 
+    def training_step(self, batch, batch_idx):
```

Comment:
could you type-hint this ? one could expect a tensor here, but it's unwrapped with ** so it's something else (dict I guess), just passing in the expected type helps understanding the convention at a glance I think